docs/learn/smart-contract-developers/_category_.yml

```diff
@@ -1,3 +1,3 @@
-label: '🧙‍♂️ Working with Smart Contracts'
+label: '🧙‍♂️ Working with Contracts'
```

Comment:
Why?

I think it is god to keep "smart contracts" (SEO, etc.)

Reply:
It's the only chapter that does not fit within one line 😅 Maybe do `Build Smart Contracts` or `Create Smart Contracts`? WDYT?

![img1](https://github.com/lukso-network/docs/assets/61689369/e2a19c3c-9005-4f81-b8d9-264fe5c257e1)